src/ctim/generators/schemas/judgement_generators.cljc

```diff
@@ -1,7 +1,7 @@
 (ns ctim.generators.schemas.judgement-generators
   (:require [clj-momo.lib.time :as time]
             [clojure.test.check.generators :as gen]
-            [ctim.schemas.common :as schemas-common]
+            [ctim.schemas.shared.common :as schemas-common]
```

Comment:
I would prefer to not add another layer of namespace hierarchy.

And instead change the name of the other ObservedRelationship or perhaps move to something like ObservedData from STIX